taesch/README.md

```diff
@@ -1,16 +1,46 @@
 # taesch
+Comfort Shopping
 
-A new Flutter project.
+<img src="https://github.com/sorrowcode/taesch-/blob/developer/taesch/assets/logo.png?raw=true" width=20% height=20%>
+
+taesch makes your shopping easier. taesch learns from your previous shopping in a supermarket and proposes you the next items to put in your bag.
 
 ## Getting Started
+Prerequisite installed git & the Flutter SDK
+
+Clone the project
+```bash
+  git clone https://github.com/sorrowcode/taesch-
+```
+
+Go to the project directory
+```bash
+  cd taesch-/taesch
+```
+
+Install dependencies
+```bash
+  flutter pub get
+```
+
+Build debug build
+```bash
+  flutter run
+```
 
-This project is a starting point for a Flutter application.
+Build release build
+```bash
+  flutter run --release
+```
 
-A few resources to get you started if this is your first Flutter project:
+## Running Tests
+To run tests, run the following command
+```bash
+  flutter test
+```
```

Comment:
interessant wäre es, die Pipeline per Docker-Container noch zu prüfen. da gäbe es

```bash
docker build .
```